vocab/properties.json

```diff
@@ -911,23 +899,23 @@
         "schemas": []
     },
     "identifier": {
-        "description": null,
+        "description": "Term or code used to identify something or someone.",
         "name": "Identifier",
         "sameAs": null,
         "schemas": [
             "core/schemas/v3/miscellaneous/digitalIdentifier.schema.tpl.json"
         ]
     },
     "identifierPattern": {
-        "description": null,
+        "description": "Format of a term or code used to identify something or someone.",
         "name": "Identifier pattern",
         "sameAs": null,
         "schemas": [
             "core/schemas/v3/miscellaneous/digitalIdentifierSchema.schema.tpl.json"
         ]
     },
     "identifierSchema": {
-        "description": null,
+        "description": "Service institution providing particular formatted identifier for something or someone.",
```

Comment:
I don't really understand this one.

Reply:
It should describe e.g. "DOI", "ORCID", "UUID" as a service. But I guess it's rather a standard?

Reply:
Would the following be better?
"Standard for creating a particular identifier for something or someone."